.werft/jobs/build/installer/installer.ts

```diff
@@ -161,6 +163,16 @@ EOF`)
         done`, { slice: slice })
     }
 
+    private configureStripeAPIKeys(slice: string) {
+        exec(
+            `kubectl --kubeconfig ${CORE_DEV_KUBECONFIG_PATH} -n werft get secret stripe-api-keys -o yaml > stripe-api-keys.secret.yaml`,
+            { slice },
```

Comment:
This naive way of dumping out the secret as `yaml` before `kubectl apply`ing only works once; the next attempt to `apply` the secret will fail due to the internal fields present in the `yaml.`

See:

https://github.com/gitpod-io/gitpod/pull/10563#issuecomment-1151361102

and a discussion of the problem eg [here](https://stackoverflow.com/questions/51297136/kubectl-error-the-object-has-been-modified-please-apply-your-changes-to-the-la) and [here](https://fabianlee.org/2022/01/26/kubernetes-alternative-to-export-for-removing-internal-fields-from-yaml/).

Reply:
Resolved by deleting metadata fields before applying.